content/en/_index.md

```diff
@@ -8,25 +8,25 @@ css: scss/index.scss
 section1:
   title: KubeSphere Container Platform
   topic: The Kubernetes Platform, tailored for the hybrid cloud
-  content: KubeSphere is a distributed operating system providing cloud native stack with Kubernetes as its kernel, and aims to be plug-and-play architecture for third-party applications seamless integration to boost its ecosystem.
+  content: KubeSphere is a distributed operating system providing cloud native stacks with Kubernetes as its kernel, and aims to create a plug-and-play architecture for third-party applications for seamless integration to boost its ecosystem.
```

Comment:
I disagree with the changes in this sentence, how about it? @rayzhou2017

Reply:
Use this one:

KubeSphere is a distributed operating system managing cloud native applications with Kubernetes as its kernel, and provides plug-and-play architecture for third-party applications seamless integration to boost its ecosystem.

Reply:
this one is ok to me

Reply:
I agree

Reply:
Agree+1